code/datums/supply_packs/food.dm

```diff
@@ -55,11 +55,140 @@
 	containername = "\improper Tomato crate"
 	group = "Food"
 
+/datum/supply_packs/soybeans
+	name = "soybeans (x20)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper soybeans crate"
+	group = "Food"
+
+/datum/supply_packs/soybeans
+	name = "cocoapod,sugarcane (x10)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper soybeans crate"
```

Comment:
\improper is used for when you wish to designate a name that uses capital letters. As this name does not use capital letters it is not required. However, it should be using capital letters as that is the standard for crate names.

Reply:
i changed crates name to for example " \improper Example Crate"